Story/RainMeadow.StoryHooks.cs

```diff
@@ -326,16 +329,28 @@ private void RainWorldGame_GoToDeathScreen(On.RainWorldGame.orig_GoToDeathScreen
         {
             if (OnlineManager.lobby != null && OnlineManager.lobby.gameMode is StoryGameMode)
             {
-                if (!OnlineManager.lobby.isOwner)
+                foreach (var playerAvatar in OnlineManager.lobby.playerAvatars.Values)
                 {
-                    if (!OnlineManager.lobby.owner.OutgoingEvents.Any(e => e is RPCEvent rpc && rpc.IsIdentical(RPCs.MovePlayersToDeathScreen))) {
-                        OnlineManager.lobby.owner.InvokeRPC(RPCs.MovePlayersToDeathScreen);
+                    if (playerAvatar.type == (byte)OnlineEntity.EntityId.IdType.none) continue; // not in game
+                    if (playerAvatar.FindEntity(true) is OnlinePhysicalObject opo && opo.apo is AbstractCreature ac)
+                    {
+                        if (ac.state.alive)
+                            RainMeadow.Debug($"still alive: {playerAvatar}");
```

Comment:
Do we still need these debug statements?

Reply:
probably not

during playtesting there was a really weird situation where host died and clients sheltered but something in SessionEnded failed (seems like it couldn't find the host's realized creature?) so it went on to call GoToDeathScreen instead

haven't managed to reproduce even with getting snagged by wormgrass and falling out of the world but i suppose this debug statement wouldn't help much anyways

Reply:
Yeah that's the one thing I was afraid of. I'm pretty sure it's possible for dead players realized creatures are deleted